app/assets/javascripts/app/responsive_table.js.coffee

```diff
@@ -20,13 +20,13 @@ class window.ResponsiveTable
     @table.find("tbody tr").each (index, row) => @add_header_to_row($(row))
 
   add_header_to_row: ($row) ->
-    $row.find("td").prepend (index) => @responsive_header(index)
+    $row.find("> td").prepend (index) => @responsive_header($row, index)
 
```

Comment:
It's from CSS, but yeah, I can add a comment. I think having a comment will also help clarify _why_ it's there in addition to what it's doing.